cms/static/sass/views/_group-configuration.scss

```diff
@@ -4,12 +4,12 @@
 
   .content-primary, .content-supplementary {
     @include box-sizing(border-box);
-    float: left;
+    @include float(left);
   }
 
   .content-primary {
     width: flex-grid(9, 12);
-    margin-right: flex-gutter();
+    @include margin-right(flex-gutter());
```

Comment:
I think in case of the RTLs it's OK to keep it in the middle, since they're atomic and won't clutter the produces CSS. What do you think?